user_manual/groupware/calendar.rst

```diff
@@ -217,6 +217,14 @@ Similar to attendees you can add rooms and resources to your events. The system
 
 .. note:: Rooms and resources are not managed by Nextcloud itself and the Calendar app will not allow you to add or change a resource. Your Administrator has to install and possibly configure resource back ends before you can use them as a user.
 
+Room availability
+~~~~~~~~~~~~~~~~~
+.. versionadded:: 4.8
+
+In the ``Resources`` section, you can now find ``Room availability``, which lists all the existing rooms. You can check the availability of each room in a manner similar to checking the free/busy status of event attendees.
+
```

Comment:
Maybe mention that this is only available id the rooms / resources app has been installed? Also, does it only list rooms or also resources?

```suggestion
If the "Calendar Rooms and Resources" app is installed on your instance, you can now find ``Room availability``  the ``Resources`` section. It lists all the existing rooms. You can check the availability of each room in a manner similar to checking the free/busy status of event attendees.

```

Reply:
> Maybe mention that this is only available id the rooms / resources app has been installed? Also, does it only list rooms or also resources?

only rooms